khbr/KH2/data/enemies.yaml

```diff
@@ -3483,10 +3495,13 @@ Luxord:
   - solo
   - needs_teleport
   - needs_rc
+  # can we add a needs MSN tag?
```

Comment:
this is done not through a tag, but by the "msn_required": true attribute which already exists

Cases where Luxords msn do not show up are due to bugs, not by design.

Reply:
Understood, hopefully these cases will be fixed.